build/win32/i18n/messages.de.isl

```diff
@@ -1,8 +1,9 @@
 [CustomMessages]
```

Comment:
You can try to copy this line's enter char and paste other lines for enter character

Reply:
I think it's ok now. The huge diff comes from
![image](https://user-images.githubusercontent.com/20074286/68586090-8c0fce00-048c-11ea-8319-9943fc5b8e1d.png)

However, <F6> and <C3><B6> is basically the same.
![image](https://user-images.githubusercontent.com/20074286/68586151-a8ac0600-048c-11ea-9046-436e66da98ae.png)

Reply:
May you should set charset as ANSI

Reply:
i think i managed to fix this